tgstation.dme

```diff
@@ -7232,4 +7232,7 @@
 #include "modular_zubbers\maps\biodome\turfs.dm"
 #include "modular_zubbers\maps\biodome\vendcation.dm"
 #include "modular_zubbers\maps\biodome\weapons.dm"
+#include "modular_zubbers\code\modules\antagonists\changeling\changeling.dm"
+#include "modular_zubbers\code\modules\antagonists\changeling\powers\mutations.dm"
+#include "modular_zubbers\code\modules\antagonists\changeling\powers\spiders.dm"
```

Comment:
Don't edit this manually, tick the files in dreammaker or in VSCode do ctrl+shift+p "dreammaker: Toggle Tick"